css/values.css

```diff
@@ -0,0 +1,336 @@
+/*
+===================================
+===================================
+Our Values
+===================================
+===================================
+*/
+
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Hero
+===================================
+*/
+.hero {
+	background: linear-gradient(
+			260deg,
+			rgba(9, 9, 9, 0.8) 46.9%,
+			rgba(9, 9, 9, 0) 86.82%
+		),
+		url("../assets/images/pages/values/hero-values.png") top/cover;
+	display: flex;
+	align-items: center;
+}
+/* values heading layout */
+.hero .wrapper {
+	display: flex;
+	justify-content: flex-end;
+}
+/* values heading sizing */
+.hero-text-container {
```

Comment:
Moving this to `global.css` in my Events PR (~ line 678). Please let me know if there are any issues.